### PR TITLE
Milestone: Portfolio setup

### PR DESCRIPTION
**Pull request by kelwilson**
**My first personal project;**

- Add linters configuration files
- Add index.html file
- Add stylesheet
- Add semantic tags and images
- Add menu button
- Update H1 and P elements
- Add H2 element with text
- Add a section with a background-image
- Update the images folder
- Add Import URL fonts
- Add meyerweb reset tools for css
- Add style to the background image
- Add style to the header element
- Add style to button-section and main-background image
**ISSUE** 👍 I notice an overflowing on my horizontal axis and I tried to fix it but I couldn't, please is there any way I can fix that?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            alt=""
+          />
+        </div>
+
+        <div class="toggle">
+          <div class="navigation">
+            <img src="images/ic_menu@2x.png" class="menu-btn" alt="" />
+          </div>
+        </div>
+        </div>
```
Comment (JuliCarracedo) [inline index.html]: - [x] We often describe the header as anything on top of the website, and it is mostly used to host the navigation features of the website. Kindly change this block to be wrapped by a `header` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            feature or website. Take a look at my works. If you like what you
+            see and have a project you need coded, don’t hesitate and contact
+            me.
+          </p>
+        </div>
+
+          <div class="btn">
+            <h2 class="start-btn">Start collaboration</h2>
+          </div>
+        </header>
```
Comment (JuliCarracedo) [inline index.html]: - [x] This block actually serves the purpose of being a `section`, a block of specific information about something. Kindly switch this block wrap from a `header` tag to a `section` tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            src="images/image_geometry_1.png"
+            class="background-image"
+            alt=""
+          />
+        </div>
+
+        <div class="toggle">
+          <div class="navigation">
+            <img src="images/ic_menu@2x.png" class="menu-btn" alt="" />
+          </div>
```
Comment (JuliCarracedo) [inline index.html]: - [x] Later in the project, you'll implement a navigation feature into this hamburger button, so this is considered a navigation bar. Kindly wrap this block with a `nav` tag instead.

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @kelwilson , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️Professional Pull Request and commit titles
✔️ Excelent replication of the Figma design
## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Hello @kelwilson 😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved ✔️
Hi @kelwilson , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**